nextcloud-startup-script.sh

```diff
@@ -93,7 +93,7 @@ else
   apt install python-pip -y
  pip install \
      --upgrade pip \
-     apt-select
+     apt-select -m up-to-date -t 5 -c
```

Comment:
This is the error sort of: http://askubuntu.com/questions/838981/local-apt-mirror-says-release-cant-be-found-but-its-there-what-am-i-missing